alvr/client_core/src/connection.rs

```diff
@@ -303,16 +303,27 @@ fn connection_pipeline(
 
                 if !stream_corrupted || !settings.connection.avoid_video_glitching {
                     if capabilities.external_decoder {
-                        let mut view_params = *ctx.last_good_view_params.read();
-                        for (timestamp, views) in &*ctx.view_params_queue.read() {
+                        let mut head_pose = *ctx.last_good_head_pose.read();
+                        for (timestamp, pose) in &*ctx.head_pose_queue.read() {
                             if *timestamp == header.timestamp {
-                                view_params = *views;
+                                head_pose = *pose;
                                 break;
                             }
                         }
+
```

Comment:
We don't expect IPD (and fov) to change, but if they do we'd get a couple frames where the image will be a mess because rendered view params and stored ones don't line up. We just accept that as an issue and ignore it?

For sending the info with every encoded frame I'd actually favor still sending over the view params to mitigate that, but that's a different discussion

Reply:
> We don't expect IPD (and fov) to change, but if they do we'd get a couple frames where the image will be a mess because rendered view params and stored ones don't line up. We just accept that as an issue and ignore it?

This is what was before the previous tracking API change. I think it's mostly a non issue. 

> For sending the info with every encoded frame I'd actually favor still sending over the view params to mitigate that, but that's a different discussion

Of course, i'd like to have the client receive the view params from the server. I plan this for the v21 protocol.